docs/tutorial-web-fork.md

```diff
@@ -0,0 +1,136 @@
+---
+id: tutorial-web-fork
+title: Forking a Gnosis Protocol web interface
+sidebar_label: Overview
+---
+
+
+This tutorial will walk you through how to create your own fork of a Gnosis Protocol web interface.
+
+In summary, this tutorial covers:
+
+
+1. Forking the [dex-react](https://github.com/gnosis/dex-react) repository
+2. Setting up a custom configuration in `custom/config.yaml`
+3. Making changes to the codebase directly or inside the custom directory
+4. Deploying your modified application
+
+Let us go through these steps one by one.
+
+
+## Forking the dex-react repository
+
+
+1. To fork the repository at [https://github.com/gnosis/dex-react](https://github.com/gnosis/dex-react), you will first need to be logged into your GitHub account.
+2. Then, you will click on the **Fork** button in the upper right corner.
+
+![](assets/fork_gnosis_dex-react.png)
+
+3. You will be redirected to your fork of **dex-react** repository at address `https://github.com/<username>/dex-react`, where `<username>` is your account username.
+4. If you have [git](https://git-scm.com/) or another version control system installed, clone the repository with:
+
+    ```git clone https://github.com/<username>/dex-react.git```
+
+5. Now that you have your own copy of **dex-react **locally, navigate to its local directory and install its dependencies by running:
+    ```
```

Comment:
Would be good to specify language for syntax highlighting, 
```
```bash  or ```sh for shelf scripts
```ts or ``js for javascript
```tsx or ```jsx for JSX
```
```suggestion
    ```bash
```